CODEOWNERS

```diff
@@ -112,6 +112,7 @@ CODEOWNERS @prestodb/team-tsc
 #####################################################################
 # Prestissimo module
 /presto-native-execution @prestodb/team-velox
+/.github/workflows/prestocpp-* @prestodb/team-velox
```

Comment:
include prestodb/committers too.  Better to be more expansive with this one.

Reply:
Good catch, thanks!